api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/struct/v1/ZeroFteReasonCode.java

```diff
@@ -0,0 +1,20 @@
+package ca.bc.gov.educ.studentdatacollection.api.struct.v1;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@SuppressWarnings("squid:S1700")
+public class ZeroFteReasonCode implements Serializable {
+  private static final long serialVersionUID = 6118916290604876032L;
```

Comment:
I saw other structs had this. I am not sure what it is for or if I should be generating this id or changing it from the other file

Reply:
@mightycox It doesn't matter - you can leave it.